README.md

```diff
@@ -1,2 +1,20 @@
-Project Title: Virtual Pet
-Project Description: MCR Codes Challenge to create a virtual pet using JavaScript
\ No newline at end of file
+# Virtual Pet 
```

Comment:
This is a nice basic README, but it is lacking in a few things. Some things that would make it much better:
- installation instructions (ie cloning the repo and what commands are available),
- what technologies are required to use it, what the application dependencies are (`JavaScript`, `Jest` and `Node` in this case)
- a description of what the app actually does and what its features are. What are the rules for how the pet is fed, exercised, how long it will live, what are the max and min values for things like hunger, fitness etc.
- how the app is tested,
- what you would do if you had more time (if applicable)
- your name

You also have multiple full size headings (the ones with a single # at them beginning). There should only be one of these and it should be the title of the project. Heres a helpful cheatsheet [guide to markdown](https://www.markdownguide.org/cheat-sheet/)